API.md

```diff
@@ -236,6 +236,13 @@ Here's the complete rundown of how files and directories are mapped to API calls
   - **`routes/index.js`** - export an array of `options` or `function(server, options)` that returns an array of `options`.
   - **`routes/route-id.js`** - export `options` or `function(server, options)` that returns `options`.  If `options` is a single route config object, the route's `config.id` will be assigned `'route-id'` from the filename if it isn't already specified.  The filename could just as easily represent a group of routes (rather than an id) and the file could export an array of route configs.
 
+#### Subscriptions (for [nes](https://github.com/hapijs/nes))
+> [`server.subscription(name, [options])`](https://github.com/hapijs/nes/blob/master/API.md#serversubscriptionpath-options)
+
+  - **`subscriptions.js`** - export an array of objects `{ path, options }` or function(server, options)` that returns an array of objects.
```

Comment:
Not a huge deal, but if you have any ideas how to disambiguate the `options` in `{ path, options }` versus `function(server, options)` then I'd be interested!

Reply:
I see that some other areas of the docs have the same small issue.

Reply:
it's a tricky one, the only thing that comes to mind is to rename the `options` in `function(server, options)` to something more explicit like `pluginOptions` but i don't know as it's the cleanest or even if it makes things clearer

Reply:
That makes sense—sounds like something to tackle on another day!  Don't worry about it for this PR.  I'll try to put some thought into it.